ubuntu-16.04/cuda/8.0/devel/cudnn5/Dockerfile.ppc64le

```diff
@@ -0,0 +1,13 @@
+FROM cuda:8.0-devel
+MAINTAINER NVIDIA CORPORATION <digits@nvidia.com>
+
+ENV CUDNN_VERSION 5
+LABEL com.nvidia.cudnn.version="5"
+
+ENV CUDNN_DOWNLOAD_SUM 51f698d468401cef2e3e2ef9bb557bd57cbeb4dca895d1d1ae8a751d090bbe39
```

Comment:
You can put that in the RUN below like we do now: https://github.com/NVIDIA/nvidia-docker/blob/master/ubuntu-16.04/cuda/8.0/runtime/cudnn5/Dockerfile#L11-L12